lib/modules/blockchain_process/gethClient.js

```diff
@@ -282,6 +282,9 @@ class GethClient {
           accountAddress = address;
         }
         if (accountAddress) {
+          if(!(self.config && self.config.account && self.config.account.password)){
+            console.warn(__("\n===== Password needed =====\nPassword for account {{account}} not found. Unlocking this account may fail. Please ensure a password is specified in config/blockchain.js > {{env}} > account > password.\n", {account: address, env: self.env}));
```

Comment:
Is `{{account}}` supposed to be a string or should it be interpolated instead?

Reply:
This is for i18n translations. Any string inside of `__(" ")` is translated. So instead of adding an interpolated string in the translation, we can add in a templated string as in this example.

Reply:
Ah, thanks for clarifying! :)